src/assets/i18n/en.json

```diff
@@ -84,6 +85,7 @@
   "login.create.assignment.description": "You need to create a profile before you can create a job",
   "login.apply.assignment.title": "Search jobs",
   "login.apply.assignment.description": "You need to create a profile before you can search for a job",
+  "start.or": "or",
```

Comment:
`ng-translate` supports that too :) https://github.com/ocombe/ng2-translate/blob/master/README.md#5-use-html-tags

And since we control the HTML 100% there is no XSS vulnerability :) 🔒